phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/installations/ViewInstallations.java

```diff
@@ -0,0 +1,169 @@
+
+        // initialising the installations lists
+        this.installations = new ExpandedList<>(this.sortedCategories, InstallationCategoryDTO::getInstallations);
+        this.filteredInstallations = new FilteredList<>(this.installations);
+        this.sortedInstallations = this.filteredInstallations.sorted(Comparator.comparing(InstallationDTO::getName));
+
+        this.filter = new InstallationsFilter<>(filteredInstallations,
+                (filterText, installation) -> installation.getName().toLowerCase().contains(filterText));
+
+        this.activeInstallations.setOnMouseClicked(event -> {
+            this.activeInstallations.deselectAll();
+            this.onInstallationSelected.accept(null);
+            event.consume();
+        });
+
+        this.sidebar = new InstallationsSidebar(activeInstallations, javaFxSettingsManager);
+        this.sidebar.bindCategories(this.sortedCategories);
+
+        // set the category selection consumers
+        this.sidebar.setOnCategorySelection(category -> {
+            filter.setFilters(category.getInstallations()::contains);
+            this.closeDetailsView();
+        });
+        this.sidebar.setOnAllCategorySelection(() -> {
+            filter.clearFilters();
+            this.closeDetailsView();
+        });
+        this.sidebar.setOnSearch(searchKeyword -> {
+            final List<InstallationCategoryDTO> installationsCorrespondingToKeywords = this.categories.stream()
+                    .filter(installationDTO -> installationDTO.getName().toLowerCase()
+                            .contains(searchKeyword.toLowerCase().trim()))
+                    .collect(Collectors.toList());
+
+            Platform.runLater(() -> this.populate(installationsCorrespondingToKeywords));
+        });
+
+        this.setSidebar(this.sidebar);
+
+        this.activeInstallations.bind(sortedInstallations);
+
+        this.setCenter(this.activeInstallations);
+
+        this.installationsPanel = new InstallationsPanel();
+
+    }
+
+    private void populate(List<InstallationCategoryDTO> categories) {
+        Platform.runLater(() -> {
+            this.categories.setAll(categories);
+            this.filter.clearAll();
+            this.sidebar.selectAllCategories();
+
+            this.closeDetailsView();
+            this.setCenter(activeInstallations);
+        });
+    }
+
+    private void showInstallationDetails(InstallationDTO installationDTO) {
+        this.installationsPanel.setOnClose(this::closeDetailsView);
+        this.installationsPanel.setInstallationDTO(installationDTO);
+        this.installationsPanel.setMaxWidth(600);
+        this.showDetailsView(this.installationsPanel);
+    }
+
+    /**
+     * adds new installation
+     * @param installationDTO new installation
+     */
+    public void addInstallation(InstallationDTO installationDTO) {
+        populate(new InstallationsUtils().addInstallationToList(this.categories, installationDTO));
```

Comment:
I think we may be talking about different lists.
Because the `ObservableList` categories must be modifiable, otherwise we wouldn't be able to set its values.